Miniscript-Template-001-3-Key-Time-Layered-Multisig.mediawiki

```diff
@@ -0,0 +1,63 @@
+Miniscript Template: 1
```

Comment:
I recommend the front matter be

MSTemplate: 1

similar to the no space file naming...